package.json

```diff
@@ -113,7 +113,7 @@
     "preact": "^7.1.0",
     "preact-compat": "^3.9.0",
     "raven-js": "^3.10.0",
-    "react-bosonic": "^1.0.0-beta5",
+    "react-bosonic": "beta",
```

Comment:
beta is not the good version.

    "react-bosonic": "1.0.0-beta6",

if you use yarn you do this:

    yarn upgrade react-bosonic

this change package.json and yarn.lock so it's better to add the correct version ;)